transforms/TileTissueNet5Class.py

```diff
@@ -12,17 +12,22 @@ class TissueTileNetTransformer(TorchTransformModel):
 
     def __init__(self, use_weights=False):
         # del kwargs['depth']
-        self.model = TissueTileNet(resnet18(), 5, activation=torch.nn.Softmax(dim=0))
-        self.class_labels = {0:'Stroma', 1:'Tumor', 2:'Glass', 3:'Necrosis', 4:'TILs'}
+        self.model = TissueTileNet(resnet18(pretrained=True), 4, activation=torch.nn.Softmax(dim=1))
+        self.class_labels = {
+                0: 'Stroma',
+                1: 'Tumor',
+                2: 'Fat',
+                3: 'Necrosis'
+                }
         self.column_labels = {0:'Classification'}
-        
-        state_dict = get_state_dict_from_git_tag("main:tissue_net_2021-01-19_21.05.24-e17.pth")
+
+        state_dict = get_state_dict_from_git_tag("1525-oncofusion-classifier:tissue_type_classifier_weights.torch")
```

Comment:
Even though we're merging to main,  I think this is still safe since we'll still have the branch available to pull. Maybe long term we'll have all of the models stored on main? Worth reviewing torch-hub best practices to double check, but good for now.